client/my-sites/current-site/index.jsx

```diff
@@ -187,12 +187,12 @@ module.exports = React.createClass( {
 			<Card className="current-site">
 				{ hasOneSite
 					? this.addNewWordPressButton()
-					: <span
-						className="current-site__switch-sites"
-						onClick={ this.switchSites }>
+					: <span className="current-site__switch-sites">
```

Comment:
Without it the overwrites needed for button are a bit more messy. Also, keeping details like the bottom border out of the button element seems more solid.